### PR TITLE
#314 Restore defaults for Ubuntu 12.04 and 14.04

### PR DESCRIPTION
**Pull request by efungsh**
Added an $operatingsystemrelease case to restore Ubuntu 12.04 and 14.04 compatitibility.
Values based off defaults originally in 9832dee0a07d82e6f2f215fce10149fbb24aa573

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -199,6 +199,28 @@
       $default_service_name                    = 'ssh'
 
       case $::operatingsystemrelease {
+        /^1[24].04$/: {
```
Comment (anders-larsson) [inline manifests/init.pp:202]: You might want to escape the dot to ensure it is an actual dot when evaluated.

**Comment by anders-larsson** [comment]
Hello. We've also been hit by this subtly removal of compatibility.  Looks good to me other the nitpicking added in comment :)

File: manifests/init.pp [hunk 2]
```diff
@@ -205,6 +205,28 @@
       $default_service_name                    = 'ssh'
 
       case $::operatingsystemrelease {
+        /^1[24]\.04$/: {
```
Comment (ghoneycutt) [inline manifests/init.pp:208]: Instead of a regex, a list would be easier to read.

```puppet
'12.04', '14.04': {
```

**Review by ghoneycutt — CHANGES_REQUESTED** [review]

**Comment by ghoneycutt** [comment]
I'm not interested in supporting these EOL platforms.

**Pull request closed without merging**